src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -3262,8 +3262,8 @@ def test_ps_s3_persistent_topic_configs_ttl():
 @attr('basic_test')
 def test_ps_s3_persistent_topic_configs_max_retries():
     """ test persistent topic configurations with max_retries and retry_sleep_duration """
-    config_dict = {"time_to_live": "None", "max_retries": 20, "retry_sleep_duration": 1}
-    buffer = 20
+    config_dict = {"time_to_live": "None", "max_retries": 10, "retry_sleep_duration": 1}
```

Comment:
how are these changes related?

Reply:
ok. but this was always the case, unrelated to this PR, right?

Reply:
yes this PR didn't change the functionality of retries